src/content/0/en/part0b.md

```diff
@@ -121,7 +121,7 @@ In traditional web applications the browser is "dumb". It only fetches HTML data
 The example uses [Express](https://expressjs.com/) from Node.js. 
 This course will use Node.js and Express to create web servers. 
 
-### Running application logic on the browser
+### Running application logic in the browser
```

Comment:
I'm unsure about this one. On one hand I'm accustomed to saying "executing code *on the server*" but on the other hand I'm also familiar with "executing code *in the browser*".